include/zephyr/modem/chat.h

```diff
@@ -45,9 +45,9 @@ struct modem_chat_match {
 	/** Size of separators array */
 	uint8_t separators_size;
 	/** Set if modem chat instance shall use wildcards when matching */
-	uint8_t wildcards : 1;
+	bool wildcards : 1;
 	/** Set if script shall not continue to next step in case of match */
-	uint8_t partial : 1;
+	bool partial : 1;
```

Comment:
No need for the ` : 1` part anymore?

Reply:
The size of a bool is 1 byte, so we save a byte by using a bitfield :)

Reply:
I wouldn't use a bitfield here as it may cause other alignment/filling issues and performance may suffer (we may burn some cycles due to masking and shifting). Is it really worth saving just a byte in that case?

Reply:
bitfield has been removed :)